src/starterbot/ProductionManager.h

```diff
@@ -13,6 +13,8 @@ class ProductionManager : public EventReceiver {
     // gathering minerals are not reserved to keep them available for other managers.
     bw::Unitset m_workers;
 
+    bw::Unitset m_gas_gatherers;
```

Comment:
Should be `m_gasGatherers`, since we use `camelCase` for variable names. The underscore is only used for the `m_` prefix.